query/rancher/rancher.go

```diff
@@ -66,6 +71,44 @@ func (r Client) GetK8sDistributions() (map[string]int, error) {
 	return distributions, nil
 }
 
+func (r Client) GetLatestRancherVersion() (map[string]int64, error) {
+	resp, err := http.Get("https://api.github.com/repos/rancher/rancher/releases/latest")
+	defer resp.Body.Close()
+	if err != nil {
+		log.Fatal("failed to open url")
+		return nil, err
+	}
+
+	bodyBytes, err := io.ReadAll(resp.Body)
+
+	if err != nil {
+		log.Fatal("failed to read HTTP body")
```

Comment:
I'm not sure fatal is what we want here? Should we be killing off this thing if we can't talk to github?

Reply:
Good point, I'm guessing we should be handling this in collector.go? I'll remove the log.fatal messages from GetLatestRancherVersion() in favour of handling the errors in collector.go